.github/workflows/osrm-backend-docker.yml

```diff
@@ -0,0 +1,84 @@
+name: build and publish container image
+on:
+  push:
+    tags:
+      - 'v*'
+    branches:
+      - master      
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:       
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ghcr.io/${{ github.repository }}/osrm-backend
```

Comment:
[github.repository](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) includes the repository name.